my-app/src/app/models/course.interface.ts

```diff
@@ -0,0 +1,23 @@
+export interface ICourse {
+    id: number;
+    title: string;
+    creationDate: string;
+    duration: number;
+    description: string;
+}
+
+export class Course implements ICourse {
```

Comment:
You can use a public keyword to make it shorter 
```
export class Course implements ICourse {
  constructor(
    public id: number,
    public title: string,
    public creationDate: string,
    public duration: number,
    public description: string
  ) {}
}

Reply:
Fixed!